.env.nice-agent-alice.dev

```diff
@@ -3,5 +3,3 @@ DB_PORT=3020
 CLOUDAGENT_PORT=3010
 PORTAL_BASE_URL=http://host.docker.internal:3000
 DATABASE_URL=postgresql://postgres:postgres@localhost:3020/nice-agent-portal
-VERITABLE_PG_HOST="localhost"
```

Comment:
I think we need these env vars for running in "development mode", where the dependencies are run through compose, but one of the agents is run on the host machine

Reply:
in dev mode `veritable-cloudagent` + its postgres still runs in docker so I don't see why the host would be different? It's only `nice-agent-portal` running on the host machine, and that doesn't talk to `veritable-cloudagent`s postgres